02LessonTwo/homework/Homework.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="header">
+    <h1>Larry Gutierrez</h1>
+    <h1>EST.1991<h1>
+  </div>
+  <div id="left">
+    <table>
+      <th>
+        <h2>About Me</h2>
+        <p>Whats up guys im Larry also know in the streets as Bird you guys can most probably make the
+        connection there. Lived all my life in San Antonio so you know im a spurs fan.
+        A entrepenuer at heart always curious about how thing come into existense, how they work
+        and grow. I like to think of my self as a discplined, honest and loving individual. Family
+        is everything to me. I love chinesse food like seriously i can probaly eat all the time.
+       </p>
+     </th>
+   </table>
+     <h3>Hobbies<h3>
+     <Ul>
+       <li>Outdoors</li>
+       <li>traveling</li>
+       <li>futbol</li>
+       <li>boxing</li>
+     </Ul>
+
+      <H3>Thing i dont like.</H3>
+      <select name="cars">
+  <option value="v">Volvo</option>
+  <option value="A">Saab</option>
+  <option value="fiat">waking up early</option>
+  <option value="terror">ISIS</option>
+  <option value="audi">alergies</option>
+  <option value="audi">cold</option>
+  <option value="audi">loosing</option>
+  <option value="audi">Audi</option>
+  <option value="audi">Audi</option>
+  <option value="audi">Audi</option>
+</select>
+</div>
```

Comment:
Try to remember to keep the same indents with the opening and closing tags. Here you moved the closing `<div>` tag back to the far left. Same goes for the `<select>` tags on lines 33 and 44.